autocannon.js

```diff
@@ -0,0 +1,15 @@
+// eslint-disable-next-line import/no-commonjs, import/no-unresolved
+const autocannon = require('autocannon');
```

Comment:
Did you mean to implement this on this branch? Is it relevant to the admin panel?